src/lib/components/molecules/waffled-squares/index.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { SvgSquare } from '$particles/svg-square'
+
+export const WaffledSquare = ({
```

Comment:
Having taking a look at the component, it strikes me that it isn't actually very different from a standard waffle chart. I realise this might be annoying, but I think perhaps that we make our waffle component handle your use case instead of creating a new component for it. @7091lapS has already made a waffle component (which is currently waiting for review, but we can get that done soon)

If we did that, we would have to remove handle titel and the black border outside of the waffle component, but I think that's the right approach. The waffle component should just draw a waffle and nothing else, otherwise it becomes much less reusable.

Reply:
So you think we could just extent Antonio´s waffle in order to add Country name, seats label and black stroke? I'm also thinking that it has to have a background as it need to be reconfigured to a choropleth map hidding the waffle when that option is selected. Perhaps this component then, should include Antonio's one plus other behaviors?